libsession/src/main/java/org/session/libsession/messaging/sending_receiving/pollers/OpenGroupPoller.kt

```diff
@@ -139,7 +139,7 @@ class OpenGroupPoller(private val server: String, private val executorService: S
             OpenGroupMessage(
                 serverID = it.id,
                 sender = it.sessionId,
-                sentTimestamp = TimeUnit.SECONDS.toMillis(it.posted),
+                sentTimestamp = TimeUnit.SECONDS.toMillis((it.posted * 1000).toLong()),
```

Comment:
Does the `TimeUnit.SECONDS.toMillis` need to still be here after the * 1000? I'm not sure what `OpenGroupMessage.fromJson()` was doing

Reply:
That's not meant to be there, thanks. I picked this up in my testing earlier and fixed it in my last commit.